build.gradle

```
@@ -47,8 +47,22 @@ subprojects {
     }
 
     dependencies {
+        def parchmentAppendix
+        def parchmentVersion
+        def parts = rootProject.parchment_version.split('-')
+        if (parts.size() == 2) {
+            parchmentAppendix = parts[0]
+            parchmentVersion = parts[1]
+        } else {
+            parchmentAppendix = rootProject.minecraft_version
+            parchmentVersion = parts[0]
+        }
```

If you don't care about having to specify parchment's mc version, we could simplify this:

```suggestion
        def parchmentAppendix = rootProject.parchment_version.split('-')[0]
        def parchmentVersion = rootProject.parchment_version.split('-')[1]
```

---

Specifying the mc version is fine, I don't think the added complexity is worth it